tests/search/search_aggregates_v8/test_aggregation/search_clients_daily_v8_bq_test.schema.json

```diff
@@ -2,102 +2,96 @@
   {
     "type": "DATE",
     "name": "submission_date",
-    "mode": "REQUIRED"
+    "mode": "NULLABLE"
   },
   {
-    "type": "INTEGER",
-    "name": "profile_creation_date",
-    "mode": "REQUIRED"
+    "type": "STRING",
+    "name": "client_id",
+    "mode": "NULLABLE"
   },
-
   {
     "type": "STRING",
-    "name": "app_version",
+    "name": "engine",
```

Comment:
modifying the test to change the column order makes the actual changes to the schema hard to indentify, and chance you could restore the original column order, or put changing the order in a separate PR?